safitty/parser.py

```diff
@@ -109,18 +112,22 @@ def load(
     if not path.exists():
         raise Exception(f"Path '{path}' doesn't exist!")
 
-    if not is_file_supported(path):
-        raise Exception(f"Unknown file format '{path.suffix}'")
+    if data_format is not None:
+        suffix = data_format
+    else:
+        suffix = path.suffix
+    assert is_file_supported(suffix), \
+        f"Unknown file format '{suffix}'"
```

Comment:
maybe it is better to use:
```suggestion:-1+0
    if not is_file_supported(suffix):
        raise Exception(f"Unknown file format '{suffix}'")
```

Reply:
agreed :)